packages/payment-processor/test/payment/index.test.ts

```diff
@@ -389,6 +408,38 @@ describe('hasSufficientFunds', () => {
     expect(spy).toHaveBeenCalledTimes(1);
   });
 
+  it('should call the ERC777 getBalance method', async () => {
```

Comment:
```suggestion
  it('should call the ERC20 getBalance method for ERC777 requests', async () => {
```
(if I understand well)